cocos/core/director.ts

```diff
@@ -172,29 +172,29 @@ export class Director extends EventTarget {
     public static readonly EVENT_BEFORE_COMMIT = 'director_before_commit';
 
     /**
-     * The event which will be triggered before the physics process.<br/>
-     * 物理过程之前所触发的事件。
+     * @en The event which will be triggered before the physics process.<br/>
```

Comment:
Remove `<br/>` check all existence in the file

Reply:
seems will be invisible in final html